test/screens/v2/candidate_generator/elevator/closures_test.exs

```diff
@@ -6,25 +6,18 @@ defmodule Screens.V2.CandidateGenerator.Elevator.ClosuresTest do
 
   alias Screens.Alerts.{Alert, MockAlert}
   alias Screens.Facilities.MockFacility
-  alias Screens.LocationContext
   alias Screens.Routes.{MockRoute, Route}
   alias Screens.Stops.{MockStop, Stop}
   alias Screens.V2.CandidateGenerator.Elevator.Closures, as: ElevatorClosures
   alias ScreensConfig.Screen
   alias ScreensConfig.V2.Elevator
 
-  describe "elevator_status_instances/5" do
-    test "Only returns alerts with effect of :elevator_closure" do
-      now = ~U[2024-10-01T05:00:00Z]
-
+  describe "elevator_status_instances/1" do
+    test "Only returns closures with effect of :elevator_closure" do
```

Comment:
These test descriptions are another instance of "closures" when we really mean "alerts".